backend/prisma/seed.ts

```diff
@@ -0,0 +1,54 @@
+// https://www.prisma.io/docs/guides/database/seed-database
+import { PrismaClient } from '@prisma/client';
+import * as argon from 'argon2';
+
+const prisma = new PrismaClient();
+
+// TODO: setup an account with 2fa enabled for testing
+async function populateUsers() {
+  await prisma.user.create({
+    data: {
+      nickname: 'gmorange',
+      ladderPoints: 682,
+      hash: await argon.hash('gmetire')
+    }
+  });
+  await prisma.user.create({
+    data: {
+      nickname: 'abourdar',
+      ladderPoints: 836,
+      hash: await argon.hash('pioupiou')
+    }
+  });
+  await prisma.user.create({
+    data: {
+      nickname: 'lpassera',
+      ladderPoints: 712,
+      hash: await argon.hash('dudududududuel')
```

Comment:
![Time to d-d-duel](https://media.tenor.com/XoYV9sqXITkAAAAC/yu-gi-oh-time-to-duel.gif)